docs/usage/etcd_member_garbage_collection.md

```diff
@@ -0,0 +1,30 @@
+## Superfluous member deletion in the ETCD cluster
+
+With support for multi-node operation, there is an expectation that the cluster size can be dynamic. In such scenarios, a reduction in the cluster size should not result in any superfluous members or entries being left over.
+
+Such superfluous members/entries can have an adverse effect on the quorum size of the cluster and could result in a degradation of service.
+As such, care has to be taken to periodically check and remove any superfluous entries.
+
+There is the possibility of superfluous entries in the `members` array of the ETCD CRD as well as in the ETCD cluster for which there is no corresponding pod in the `StatefulSet` anymore.
+Keeping the `members` array on the ETCD CRD in check is the responsibility of `etcd-druid` whereas keeping the ETCD cluster in check is the responsibility of the leading `etcd-backup-restore` sidecar.
+The purpose of this document is the responsibility of the leading `etcd-backup-restore` sidecar to keep the ETCD cluster in check
+
+
+#### Example
+Below is an example of how we could end up with superfluous members in the ETCD cluster.
+This is a common scenario that could occur frequently and demonstrates the need for a garbage collection mechanism that detects and clears such superfluous entries automatically
+```
+Suppose an ETCD cluster is scaled up from `3` to `5` and the newly added members are failing constantly due to insufficient resources and then if the ETCD cluster is scaled back down to `3`. The failing member pods may not have the chance to clean up their `member` entries from the ETCD cluster leading to superfluous members in the ETCD cluster that may have an adverse effect on the quorum of the cluster.
```

Comment:
```suggestion
Suppose an etcd cluster is scaled up from `3` to `5` replicas, and the newly added members are failing to come up due to insufficient resources. If the etcd cluster is then scaled back down to `3` replicas, the failing member pods may not have the chance to clean up their own `member` entries from the etcd cluster, resulting in superfluous members in the etcd cluster. This directly affects the quorum of the cluster, since failure of either of the 3 existing members will lead to a quorum loss, since the cluster size is still `5` according to the etcd cluster state.
```